README.md

```diff
@@ -311,18 +311,24 @@ to the lsp client or server you are using.
 
 ### Server
 
-rust-analyzer is the default and can be changed to rls. lsp-mode
-related code was moved to the lsp-mode repo.  `rustic-lsp-server` sets
-the value of `lsp-rust-server`.
+rust-analyzer is the default and can be changed to rls if required (Note that rls
+is deprecated and is slated to be removed). lsp-mode related code was
```

Comment:
s/slated/stated/ ?

Reply:
I do mean `slated` to indicate that any feature related to rls is expected to be removed in future.